README.md

```diff
@@ -40,8 +40,8 @@ All of our forums, chat rooms, and issue trackers are governed by our [Code Of C
 
 In addition to that, you may enjoy following:
 
-- The [news](http://akka.io/news) section of the page, which is updated whenever a new version is released
-- The [Akka Team Blog](http://blog.akka.io)
+- The [news](https://akka.io/news) section of the page, which is updated whenever a new version is released
+- The [Akka Team Blog](https://blog.akka.io)
```

Comment:
... and this redirects to https://akka.io/blog/

Reply:
ok, let me fix that